libraries/botbuilder-ai/src/qnaMaker.ts

```diff
@@ -180,11 +180,17 @@ export class QnAMaker implements QnAMakerClient, QnAMakerTelemetryClient {
     }
 
     // Gets a value indicating whether determines whether to log personal information that came from the user.
+    /**
+     *@returns True if will log personal information into the BotTelemetryClient.TrackEvent method; otherwise the properties will be filtered.
```

Comment:
```suggestion
     * @returns True if will log personal information into the BotTelemetryClient.TrackEvent method; otherwise the properties will be filtered.
```